evergreen/src/e/edit/ExternalTools.java

```diff
@@ -128,7 +144,10 @@ private static void parseFile(File file, List<ExternalToolAction> newAllTools, L
         boolean needsFile = false;
         boolean showOnPopUpMenu = false;
         
-        for (String line : StringUtilities.readLinesFromFile(file)) {
+        for (String line : Files.readAllLines(file)) {
+            if (line == "" || line.startsWith("#")) {
```

Comment:
`line == ""` looks like a bug - i don't think comparing references is the right thing to do here.